blivet/threads.py

```diff
@@ -33,7 +34,7 @@
 
 def exclusive(m):
     """ Run a callable while holding the global lock. """
-    @wraps(m)
+    @wraps(m, set(functools.WRAPPER_ASSIGNMENTS) & set(dir(m)))
```

Comment:
Interesting. I cannot see anything like that mentioned in the documentation. How / from where did you get the idea that something like this is necessary?

This surely is no big deal, I'm okay with this being merged as it is. :)